tests/src/test/scala/whisk/core/cli/test/ApiGwTests.scala

```diff
@@ -49,10 +50,8 @@ class ApiGwTests
     // throttling restriction.  To avoid CLI failures due to being throttled, track the
     // CLI invocation calls and when at the throttle limit, pause the next CLI invocation
     // with exactly enough time to relax the throttling.
-    val throttleWindow = 1.minute
-    var cliCallCount = 5  // Set to >0 to allow for other action invocations in prior tests
-    var clearedThrottleTime = Instant.now
     val maxActionsPerMin = WhiskProperties.getMaxActionInvokesPerMinute()
+    val invocationTimes = new ArrayBuffer[Instant]()
```

Comment:
You can use `val invocationTimes = Buffer[Instant]()`. The actual impl. doesn't matter in this case.